src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/tools/SparkSstFileGenerator.scala

```diff
@@ -37,7 +37,7 @@ object SparkSstFileGenerator {
   val SSF_OUTPUT_HDFS_DIR_CONF_KEY = "nebula.graph.spark.sst.file.hdfs.dir"
 
   /**
-    * cmd line's options, which's name following the convention: input will suffix with "i", output will suffix with "o"
+    * cmd line's options, whose name follow the convention: input will start with suffix "i", output with "o"
```

Comment:
I really mean suffix "i",  not `start with suffix i`

Reply:
sorry but 'suffix' is not a verb

Reply:
`start with suffix xxx` sounds weird,  People would be wondering which option name convention should be followed:
_xxx_someOption OR someOption_xxx_ (Seriously, i means the latter). 
How about put it this way:
`input options name end with 'i', while output end with 'o'`

Reply:
good idea.